internal/infrastructure/gateways/vincario_api_service.go

```diff
@@ -196,3 +198,28 @@ func (v *VincarioInfoResponse) GetStyle() string {
 func (v *VincarioInfoResponse) GetSubModel() string {
 	return strings.TrimSpace(v.Body)
 }
+
+// GetMetadata returns a map of metadata for the vehicle, in standard format.
+func (v *VincarioInfoResponse) GetMetadata() null.JSON {
+
+	metadata := map[string]interface{}{
+		"fuel_type":              v.FuelType,
+		"driven_wheels":          v.Drive,
+		"number_of_doors":        v.NumberOfDoors,
+		"base_msrp":              nil,
+		"epa_class":              v.EmissionStandard,
+		"vehicle_type":           v.Body,
+		"mpg_highway":            nil,
+		"mpg_city":               nil,
+		"fuel_tank_capacity_gal": nil,
+		"mpg":                    v.EngineDisplacement,
```

Comment:
this one set to nil, i don't think vincario has this. Maybe fuel consumption?

Reply:
vincario does not have a fuel consumption property so I'm setting this to nil.